dashboard/app/models/course_version.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: course_versions
```

Comment:
where will the name `csp-2019` live -- is that going to be the name of the unit group?

Reply:
I think this would work -- the course version could just delegate the `name` method to its content root, pulling in either the unit group name or the unit name.

Reply:
> where will the name `csp-2019` live -- is that going to be the name of the unit group?

This isn't totally clear to me yet. In the ideal world, do we even need that name? Wouldn't we just need the two pieces of info it contains, which are the course name `csp` and the version name `2019`?

Tentatively I'm thinking that we can continue to have `csp-2019` live on the unit group, while moving towards the goal of not depending on it.

Another option could be to just have `csp-2019` be the key of the course version. Thoughts?

Reply:
> Tentatively I'm thinking that we can continue to have csp-2019 live on the unit group, while moving towards the goal of not depending on it.

This sounds right to me. It'll take some work to move off of this pattern, which makes me not want to do it right now. we'll already be using the pattern of delegating methods to the content_root, so doing this for `name` should fit into that pattern well.